packages/data/src/api/tokenById/tokenById.types.ts

```diff
@@ -0,0 +1,55 @@
+export interface NftTokenData {
```

Comment:
Is this redundant to the generated graphql options?

Reply:
no the GraphQl generated is wrong it has props like listings, that if you put on the query will break it